script/Ciel/Day 14/14_01B_02_CIEL14B_2B.txt

```diff
@@ -526,32 +643,38 @@ creating a shitty situation that cannot be controlled."
 [sha:2b98c6fce956386a1c26dad6df492ced6e3fbb75]{
 -- Page 59, Offset 39541.
 -- そんなの見れば分かるけどな！」
-I can understand that much just from looking!"
+And hey, I can tell that much just by looking at it!"
 }
 [sha:19c54618671fbbb9e287ad57d830d6ef3d0e2b75]{
 -- Page 59, Offset 39542.
 -- そうだ。問題はアルクェイドだけじゃない。
-That's right. The problem isn't just Arcueid.
+Besides, Arcueid isn't the only problem here.
 }
 [sha:26288d60ccc4d5d5a5d68b18cef6309bc5023bae]{
 -- Page 59, Offset 39543.
 -- あの光の中で、シエル先輩はどうなっている―――！？
-What's happening to Ciel-senpai inside of that light―――?!
+What's happening to Ciel-senpai inside that light―――?!
```

Comment:
Not feeling it. Overused form of それより outta nowhere, loses the progressive?

Reply:
IIRC, I didn't like how it continued from the previous line. Specifically because it continues into a question.
My suggestion is quite bad, looking at it now, but partially it just doesn't go far enough. 

>How Ciel is holding up inside that light is more pressing to me!

Well, to give one suggestion.